dashboard/modules/serve/serve_agent.py

```diff
@@ -110,11 +110,20 @@ async def delete_serve_application(self, req: Request) -> Response:
 
         return Response()
 
+    @routes.delete("/api/serve/applications")
```

Comment:
```suggestion
    @routes.delete("/api/serve/applications/")
```
Also update the `PUT` endpoint. This is to match existing deployments API (and REST API standard practice AFAIK)